Source/GB.tnLabs.Web/API_Controllers/ManageController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Http;
+using System.Text.RegularExpressions;
+using GB.tnLabs.Core.Components;
+
+namespace GB.tnLabs.Web.API_Controllers
```

Comment:
the folder name and the namespace name should be APIControllers not API_Controllers. (Bad practice)